cogs/welcome/welcome.py

```diff
@@ -529,3 +548,123 @@ async def test(self, ctx: Context):
+            descDict[userId] = description
+        # due to how pagify works, warn if there is a line feed character in the description
+        if "\n" in description:
+            await ctx.send(
+                warning(
+                    "The description has line feeds (line breaks). "
+                    "This may cause formatting issues with pagination. "
+                    "Consider removing them."
+                )
+            )
+        await ctx.send(
+            info(f"Description set for {user.mention}."),
+            allowed_mentions=discord.AllowedMentions.none(),
+        )
+        LOGGER.info(
+            "A welcome description has been added for %s#%s (%s)",
+            user.name,
+            user.discriminator,
+            user.id,
+        )
+        LOGGER.debug(description)
+
+    # [p]welcome tag remove
+    @tag.command(name="remove", aliases=["delete", "del", "rm"])
+    async def removeTag(self, ctx: Context, user: discord.User):
+        """Remove a description from a user.
+
+        Parameters:
+        -----------
+        user: discord.User
+            The user to remove a description from.
+        """
+        userId = str(user.id)
+        async with self.config.guild(ctx.guild).get_attr(KEY_DESCRIPTIONS)() as descDict:
+            if userId in descDict:
+                del descDict[userId]
+        await ctx.send(
+            info(f"Description removed for {user.mention}."),
+            allowed_mentions=discord.AllowedMentions.none(),
+        )
+        LOGGER.info(
+            "A welcome description has been removed for %s#%s (%s)",
+            user.name,
+            user.discriminator,
+            user.id,
+        )
+
+    # [p]welcome tag list
+    @tag.command(name="list", aliases=["ls"])
+    async def listTags(self, ctx: Context):
+        """List all descriptions."""
+        currentGuild: discord.Guild = ctx.guild
+        descDict: dict = await self.config.guild(currentGuild).get_attr(KEY_DESCRIPTIONS)()
+        if not descDict:
+            await ctx.send(info("No descriptions have been added."))
+            return
+        pageList = await createTagListPages(
+            descDict, embedTitle=f"Welcome descriptions for {currentGuild.name}"
+        )
+        await menu(ctx, pageList, DEFAULT_CONTROLS)
+
+    # [p]welcome tag get
+    @tag.command(name="get", aliases=["show"])
+    async def getTag(self, ctx: Context, user: discord.User):
+        """Get a description for a user.
+
+        Parameters:
+        -----------
+        user: discord.User
+            The user to get a description for.
+        """
+        userId = str(user.id)
+        descDict: dict = await self.config.guild(ctx.guild).get_attr(KEY_DESCRIPTIONS)()
+        if userId in descDict:
+            description = descDict[userId]
+            if description:
+                descText = "\n".join(
+                    [
+                        "**{}:**".format(user.mention),
```

Comment:
I initially did it just because I wanted the opening double-quotes for strings that have variables in them to also line up with the opening double-quotes for strings that do not have variables in them, because at the time I manually did `` ``` `` instead of using `box()` from `redbot`. It looked like this:
```
                    [
                        "**{}:**".format(user.mention),
                        "```",
                        "{}".format(description),
                        "```",
#                       ^
#                       |
#                       | these opening double-quotes line up
```
Nothing other than that. Can do f-string.